apps/service_define/config/dev.exs

```diff
@@ -1,5 +1,20 @@
 use Mix.Config
 
+config :service_define, Define.Application,
+  kafka_endpoints: nil,
```

Comment:
Do you need this line?

Reply:
we shouldn't; it's not explicitly defined in `config.exs` so it should default to `nil`